### ISSUE TITLE
A11y_HDInsightDevTools-BM-June2020_HDInteractive:PysparkInteractive_Keyboard : Keyboard focus is not visible for several controls in 'Python Interactive' screen.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode-jupyter#474]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."**

### Environment Details:
Appliaction Name : HDInsight Dev Tools
Visual Studio Code Version details
Version: 1.46.0 (user setup)
Commit: a5d1cc28bb5da32ec67e86cc50f84c67cc690321
Date: 2020-06-10T09:03:20.462Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
Python Extension: Version 2020.4.76186
Spark & Hive Tools Extension: Version 1.1.10
Standard Violated: WCAG2.4.7

### Repro Steps:
1. Download and install HDInsights for VS code.
2. Open any python file and link it with the cluster using proper credentials.
3. Select the code-->right click-->HDInteractive: Pyspark Interactive.
4. Using keyboard navigate to the controls such as "Gather Code", "Go to Code", "Remove Cell", etc. and observe if the keyboard focus is clearly visible or not.

### Actual:
When keyboard users navigate through the 'Python Interactive' screen several controls such as "Gather Code", "Go to Code", "Remove Cell", etc. receive keyboard focus but the keyboard focus is not visible for the users.

This issue is observed in both default light and dark themes.

### Expected:
The keyboard focus for the controls present in the 'Python Interactive' screen should be clearly visible.

### User Impact:
If the keyboard focus is not clearly visible for the controls present on the screen then the keyboard users will find it difficult to understand the navigation of the screen.

### Attachment:
[A11y_HDInsightDevTools-BM-June2020_HDInteractivePysparkInteractive_Keyboard  Keyboard focus is not visible for several controls in 'Python Interactive' screen..zip](https://github.com/microsoft/vscode-python/files/5116182/A11y_HDInsightDevTools-BM-June2020_HDInteractivePysparkInteractive_Keyboard.Keyboard.focus.is.not.visible.for.several.controls.in.Python.Interactive.screen.zip)
